requirements-test.txt

```diff
@@ -0,0 +1,9 @@
+# This is for running tox
+django-allauth
+drf-oidc-auth
+django
+pytest
+pytest-cov
+pytest-django
+pytest-mock
+responses
```

Comment:
There's already `requirements-dev.in` with the exact same contents. And purpose. Could that be used instead of adding a duplicate file?

Reply:
I considered using this, yes, but this requires compiling this for every single virtualenv that tox generates, which makes it a bit more cumbersome. And while this is possible, there's an issue with Python 3.6; it doesn't have the latest `pip-tools`, and the older version uses a legacy resolver which causes it to fail find correct versions for some of the packages (I think it was `djangorestframework`?). Also, using `pip-compile` doesn't really offer any advantages over plain old `pip` in this scenario; it basically does the same, but with an extra step.

I can use `requirements-dev.in` for this, but I don't see any point in it. IMO, it would only make sense if we used it to generate pinned versions for each environment or something for consistency's sake, but using `pip-tools` for the sake of using `pip-tools` doesn't make much sense. The other option is to use `requirements-dev.in` as a requirements file, but that would break consistency (they're usually `.txt`, not `.in`).

(As a side note, I noticed that this does have an extra line as well (`django`; we already list it as a dependency in `tox.ini`))

Reply:
For reference on what I mean with the generation thingy: [django-cors-headers](https://github.com/adamchainz/django-cors-headers/tree/main/requirements) does something similar.

Reply:
Yeah, pip-tools isn't probably much help in this project, since there are no fixed requirements (.txt) files included. Perhaps `requirements-dev.in` can be removed. And the testing instructions in `README.md` changed to use the `requirements-test.txt` file. How that sounds?

Reply:
An excellent suggestion!